kubernetes/silo-import/src/silo_import/config.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+import json
+import os
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, Optional
+
+
+@dataclass(frozen=True)
+class ImporterConfig:
+    backend_base_url: str
+    lineage_definitions: Optional[Dict[str, str]]
```

Comment:
In my understanding this should be a dictionary from version to url 
```suggestion
    lineage_definitions: dict[int, str] | None
```